talk/owt/sdk/include/cpp/owt/base/publication.h

```diff
@@ -30,10 +30,14 @@ class Publication {
   virtual void Unmute(TrackKind track_kind,
                       std::function<void()> on_success,
                       std::function<void(std::unique_ptr<Exception>)> on_failure) = 0;
-  /// Get conneciton stats of current publication
+  /// Deprecated. Get conneciton stats of current publication
```

Comment:
Please add `OWT_DEPRECATED` for deprecated functions.

Reply:
done.